.mergify.yml

```diff
@@ -1,17 +1,25 @@
 pull_request_rules:
-  - name: automatically merge Scala Steward PRs on CI success
+  - name: assign and label scala-steward's PRs
     conditions:
       - author=scala-steward
-      - base=master
-      - status-success=ci
-      - "body~=(labels: library-update, semver-minor)|(labels: library-update, semver-patch)|(labels: sbt-plugin-update, semver-minor)|(labels: sbt-plugin-update, semver-patch)|(labels: scalafix-rule-update)|(labels: test-library-update)"
     actions:
-      merge:
-        method: merge
+      assign:
+        users: [eikek]
```

Comment:
Is this necessary for some workflow? it assigns myself to all scala-steward prs?

Reply:
Not really, if you don't want to. But some PR might not get merged (for whatever reason), so you may want to be notified of it :wink: